backend/core/src/cron/cron.service.ts

```diff
@@ -1,10 +1,47 @@
 import { Injectable } from "@nestjs/common"
 import { Cron, CronExpression } from "@nestjs/schedule"
+import { UserListQueryParams } from "src/auth/dto/user-list-query-params"
+import { UserService } from "src/auth/services/user.service"
+import { AuthContext } from "src/auth/types/auth-context"
+import { Compare } from "src/shared/dto/filter.dto"
+import { EmailService } from "src/shared/email/email.service"
+import { ListingsService } from "../listings/listings.service"
 
 @Injectable()
 export class CronService {
-  @Cron(CronExpression.EVERY_WEEK)
-  handleCron() {
-    // TODO: add a cron task here.
+  constructor(
+    private readonly emailService: EmailService,
+    private readonly listingsService: ListingsService,
+    private readonly userService: UserService
+  ) {}
+
+  @Cron(CronExpression.EVERY_1ST_DAY_OF_MONTH_AT_NOON)
+  async handleCron() {
+    if (!process.env.SEND_NOTIFICATIONS_FOR_UPDATE_LISTINGS_REMINDER) {
```

Comment:
Can you add docs for this somewhere? Wiki is prolly fine.

Reply:
Yup, will do this later this week! Adding a bug for it